### DISCUSSION TITLE
More responsive admin layout

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#364] [Show and tell]
Can I propose a modified CSS to make the central part containing the list of sites responsive?
(if so, how do you do it with GitHub?)

If so, I would use the breakpoint measurements from the latest Bootstrap.

![IMG_20231205_154217.jpg](https://github.com/akeeba/panopticon/assets/35969533/2b328227-27ea-4ec8-90c8-0149c4f73463)

**Comment by nikosdion** [comment]
[Everything is SCSS](https://github.com/akeeba/panopticon/wiki/Custom-CSS). If you do a PR to the SCSS I'll compile it to CSS and if it looks right I'll merge it.
